src/Phinx/Console/Command/ListAliases.php

```diff
@@ -68,15 +68,17 @@ function ($alias, $class) use ($maxAliasLength, $maxClassLength) {
                 ),
                 $this->verbosityLevel
             );
-        } else {
-            $output->writeln(
-                sprintf(
-                    '<error>No aliases defined in %s</error>',
-                    Util::relativePath($this->config->getConfigFilePath())
-                )
-            );
+
+            return self::CODE_SUCCESS;
         }
 
-        return self::CODE_SUCCESS;
+        self::getErrorOutput($output)->writeln(
+            sprintf(
+                '<error>No aliases defined in %s</error>',
+                Util::relativePath($this->config->getConfigFilePath())
+            )
+        );
+
+        return self::CODE_ERROR;
```

Comment:
I'm not sure not having any aliases would constitute the command failing per se, should just mean that you've not set any up. I'd leave this as always returning `self::CODE_SUCCESS`.

Reply:
I'd be OK with that if the message would be downgraded to a warning or info; otherwise it'll be inconsistent (and as I wrote, my goal was to achieve consistency).

Reply:
Symfony does not ship with a warning style out of the box for the console, so I'm fine downgrading it to use `<info>` instead, or I guess could do similar to this:

https://github.com/cakephp/phinx/blob/efd7cf8b8fe3ed3ce0a6bb77fd079bcb5b732026/src/Phinx/Console/Command/AbstractCommand.php#L436

where we wrap a "warning" text in `<comment>` so it gives a bit of visual separation to the user.

Reply:
Implemented as suggested